dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/FullyQualifiedMethodSignature.java

```diff
@@ -41,6 +41,10 @@ public String getClassName() {
     return className;
   }
 
+  public MethodSignature getMethodSignature() {
```

Comment:
From the class names, I would expect that FullyQualifiedMethodSignature is a subclass of MethodSignature. Moot if we don't need either of these classes.

Reply:
Initially,

- MethodSignature represents an entry in "method" in .class file. (the source)
- FullyQualifiedMethodSignature represents an entry in Methodref in constant pool. (the targets)

I thought it would be beneficial for some point, but it's not as now.